tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/ClientSideEncryptionTestRunner.cs

```diff
@@ -207,6 +207,11 @@ private AutoEncryptionOptions ConfigureAutoEncryptionOptions(BsonDocument autoEn
             return autoEncryptionOptions;
         }
 
+        private string GetEnvironmentVariableOrDefaultOrThrowIfNothing(string variableName, string defaultValue = null) =>
```

Comment:
I don't see a significant difference but the current way looks more clear regarding what happens in the throw step. So since it's a nit I would leave it as is